### PR TITLE
vote: prompt pingyu as TiKV reviewer

### PR DESCRIPTION
**Pull request by iosmanthus**
Signed-off-by: iosmanthus <myosmanthustree@gmail.com>

### CODE REVIEW

**Review by skyzh — APPROVED** [review]
Congrats!

File: teams/tikv/team.json [hunk 1]
```diff
@@ -74,7 +74,8 @@
         "wshwsh12",
         "Xuanwo",
         "you06",
-        "zyguan"
+        "zyguan",
+        "pingyu"
```
Comment (andylokandy) [inline teams/tikv/team.json:78]: Please arrange it according to alphabet order.

**Review by andylokandy — APPROVED** [review]

**Comment by andylokandy** [comment]
+1 Congratulations!

File: teams/tikv/team.json (hunk 1 above)
Comment (iosmanthus) [reply teams/tikv/team.json:78]: Here is the output of reviewers.sort():
```
[
        "3pointer",
        "Fullstop000",
        "Jibbow",
        "Mossaka",
        "MrCroxx",
        "Xuanwo",
        "ekexium",
        "fredchenbj",
        "glorv",
        "gozssky",
        "hi-rustin",
        "jayzhan211",
        "lhy1024",
        "longfangsong",
        "lzmhhh123",
        "nolouch",
        "pingyu",
        "rleungx",
        "tier-cap",
        "wjhuang2016",
        "wshwsh12",
        "you06",
        "zyguan"
]
Should I use this as the list?

**Review by andylokandy — APPROVED** [review]

**Pull request merged**